workspace-manager-clienttests/README.md

```diff
@@ -38,13 +47,13 @@ The version of the Workspace Manager client JAR file is specified in the build.g
 fetched from the Broad Institute Maven repository. You can override this to use a local version of the Workspace Manager client
 JAR file by specifying a Gradle project property, either with a command line argument
 
-`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="configs/BasicUnauthenticated.json"
+`./gradlew -Pworkspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar runTest --args="suites/BasicIntegration.json"
 
 or an environment variable.
 
 ```
 export ORG_GRADLE_PROJECT_workspacemanagerclientjar=~/terra-workspace-manager/workspace-manager-client/build/libs/workspace-manager-client-0.5.0-SNAPSHOT.jar
-./gradlew runTest --args="configs/BasicUnauthenticated.json /tmp/TestRunnerResults"
+./gradlew runTest --args="suites/BasicIntegration.json /tmp/TestRunnerResults"
```

Comment:
Nit: Same as above. I'd just keep the single test config run command.

Reply:
Updated. With the added sections on individual versus test suite modes. It should be clear which mode we are using as an example here,